### PR TITLE
QRadar offense per call is undefined when instance already exist

### PR DESCRIPTION
**Pull request by Spivaka**
fixes https://github.com/demisto/etc/issues/11781

### CODE REVIEW

File: Integrations/integration-QRadar.yml [hunk 1]
```diff
@@ -64,7 +64,7 @@ script:
     var server = params.server+'/';
     var insecure = params.insecure;
     var query = params.query;
-    var offensesPerCall = parseInt(params.offensesPerCall);
+    var offensesPerCall = parseInt(params.offensesPerCall || 50);
```
Comment (omercnet) [inline Integrations/integration-QRadar.yml]: approved, but why is this needed? there's a default value for this parameter

Comment (dorsha) [reply Integrations/integration-QRadar.yml]: For already configured instances. Must always think about upgrade.

Comment (omercnet) [reply Integrations/integration-QRadar.yml]: right, that's why the default is there
if there's no configured value, the default should be used - why is it not being used after an upgrade?

Comment (dorsha) [reply Integrations/integration-QRadar.yml]: Because that's the way it works at the moment

Comment (dorsha) [reply Integrations/integration-QRadar.yml]: https://demisto.slack.com/archives/C1X1QDL6B/p1527163184000039

Comment (omercnet) [reply Integrations/integration-QRadar.yml]: so let's fix it :)
